docs/proposals/emptydir-pv.md

```diff
@@ -0,0 +1,151 @@
+# Emptydir persistent volume
+
+This document proposes a model for managing emptydir persistent volumes backed by node local storage.
+
+## Abstract
+We currently have:
+* Various different types of "local volumes", like emptyDir, hostPath and gitRepo
+* A way to "claim" storage, such that the pod referencing the claim can continue using the storage across pod/node restarts
+* A way to dynamically provision volumes to match un-bound/pending claims
+
+The second and third features allow us to say: provision a volume on demand, that survives pod/node restarts by using network attached storage (NFS, network block device from cloudprovider, distributed file system like gluster). This proposal is about a new `emptyDir` persistent volume that combines all three features. The idea is that we can sacrifice some durability to get a similar abstraction using *node local storage*. We expect this new PV source to be useful to applications that:
```

Comment:
It's also useful on platforms that simply don't have networked storage, regardless of the application's characteristics.